lib/school_house/content/lesson.ex

```diff
@@ -62,7 +62,7 @@ defmodule SchoolHouse.Content.Lesson do
       |> String.downcase()
 
     name
-    |> String.replace(~r/[^\w\s]/, "")
+    |> String.replace(~r/[^\p{L}\s]/u, "")
```

Comment:
@jaimeiniesta good catch on the ids! I updated this regex to handle the unicode characters and use the same `\p{L}` regex, now the links appear to be correct e.g. https://school-house-pr-122.herokuapp.com/es/basics/documentation#documentaci%C3%B3n-en-linea

Reply:
Cool! I just checked and it's fixed now. W3C Validator doesn't complain about ids anymore 🚀